new/LIT6955Lamangesta.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6955Lamangesta" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለመንግሥተ፡ ሰማይ፡ ገነት፡ ዘመዓድም፡ ጽጌሃ፤</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-mangǝśta samāy gannat za-maʿāddǝm ṣǝgehā,...  (Qǝne of ʿǝṭāna mogar type)</title>
+                <title xml:lang="en" corresp="#t1">For the Kingdom of Heaven, paradise, of its flowers being pleasant,...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation notBefore="1700" notAfter="1900"/>
+            <abstract>
+                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign>,
```

Comment:
add keyword PoemQene

should we not introduce and use keywords also for the types?

Reply:
Yes, we should; for the types and perhaps also for the cases when the type is undefined/unknown.